rocAL_pybind/rocal_pybind.cpp

```diff
@@ -129,6 +129,20 @@ py::object wrapperRocalExternalSourceFeedInput(
     return py::cast<py::none>(Py_None);
 }
 
+    py::object wrapper_one_hot_label_copy(RocalContext context, py::object p , unsigned num_of_classes, RocalOutputMemType dest_mem_type) {
+        auto ptr = ctypes_void_ptr(p);
```

Comment:
please pass as (void*) from high level and remove redundant function

Reply:
removed different functions for gpy vs the cpu case. Single function handles both